content/en/docs/concepts/security/hardening-guide/authorization-hardening.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Hardening Guide - Authorization 
+description: >
+  Information on authorization options in Kubernetes and their security properties.
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+# Authorization
+
+Authorization in Kubernetes ecosystem means specifying the privileges/permissions for a particular kubernetes resource. Defining a proper
+authorization policy is an important step in preventing unauthorized access to the clusters or privilege escalation attacks. Kubernetes offers
+multiple ways to enforce permissions which are represented by various authorization modes and modules - Node authorization, ABAC, Webhook,
+RBAC. Using the RBAC mode is the most standard way of designing permissions for cluster users and the workloads. We can set the authorization
+mode using the `--authorization-mode` flag in kube api server. A proper Authorization design helps in reducing the blast radius of any kind of
+attack.
+
+<!-- body -->
+
+### RBAC (Role Based Access Control)
+
+RBAC mode is enabled by default and is one method to control access to cluster resources based on the roles of individuals within an
+organization. It is the most important authorization method for both developers and admins in Kubernetes. It is used to restrict access for
+user accounts and service accounts. To check if RBAC is enabled in a cluster using kubectl, execute `kubectl api-version`. The RBAC mode can be
+enabled by setting the RBAC value with --authorizationmode flag in kube api server command.
+
+> `kube-apiserver --authorization-mode=RBAC`
+
+### Least Privilege Principal
+
+- The principal of least privilege should always be followed while designing the authorization policies. This principle states that any user or
+  entity should have only the required privileges/rights to access any particular resource.
+
+- Each group should only have access to the resources that they need and should not be allowed to view or alter the resources of other groups.
+  Users, user groups, and service accounts should only be able to interact with and access resources within certain namespaces.
+
+- We should never use _AlwaysAllow_ value in the authorization-mode flags, since it basically disables all the authorization modes, limiting
+  the ability to enforce the least privilege for access.
+
+- Using the _system:masters_ group for assigning rights should always be avoided, since this group has hard-coded cluster-admin rights which
```

Comment:
```suggestion
- Using the `system:masters` group when assigning rights should always be avoided, since this group has hard-coded cluster-admin rights which
```